terraform-plans/configs/charm-advanced-routing_main.tfvars

```
@@ -2,6 +2,11 @@ repository             = "charm-advanced-routing"
 repository_description = "This subordinate charm allows for the configuration of policy routing rules on the deployed host, as well as routes to configured services."
 branch                 = "main"
 templates = {
+  contribution = {
+    source      = "./templates/github/CONTRIBUTION.md.tftpl"
+    destination = "CONTRIBUTION.md"
```

CONTRIBUTING is the standard name here, and the name we use for the file across our projects.

```suggestion
    source      = "./templates/github/CONTRIBUTING.md.tftpl"
    destination = "CONTRIBUTING.md"
```